Bootloader/Adapters/Inc/hash_adapter.h

```diff
@@ -40,7 +40,7 @@
 #include "board_info.h"
 
 void HashAdapter_getHashedBoardId(uint8_t hashed_board_id[HASHED_BOARD_ID_SIZE]);
-bool HashAdapter_getBase64HashedBoardId(uint8_t* b64_hashed_board_id);
+bool HashAdapter_getBase64HashedBoardId(uint8_t* b64_hashed_board_id, size_t size);

```

Comment:
Parameter names are not the same as they are in definition. It is not mandatory to be, but in my opinion, it would be nicer to have same parameter names.

Reply:
Thanks! I also prefer to have the same name. I was using eclipse refactoring but it seems it didn't change it properly.